ts/screens/authentication/LandingScreen.tsx

```diff
@@ -293,17 +294,15 @@ class LandingScreen extends React.PureComponent<Props, State> {
           </ButtonDefaultOpacity>
           <View spacer={true} />
 
-          <ButtonDefaultOpacity
-            block={true}
-            small={true}
-            transparent={true}
-            onPress={this.navigateToSpidCieInformationRequest}
-          >
-            <Text style={styles.noPadded} link={true}>
+          <ButtonDefaultOpacity block={true} small={true} transparent={true}>
```

Comment:
This is redundant as well, but mind the style! Once you remove it, the link will lose some formatting.
Please check it.

with:
![Screenshot 2021-07-22 at 17 03 09](https://user-images.githubusercontent.com/2094604/126661970-4dcdb85d-b6c7-4000-8e6f-c479dfa2352f.png)

without:
![Screenshot 2021-07-22 at 17 03 01](https://user-images.githubusercontent.com/2094604/126661993-6425e31c-cb51-4f8f-996c-1b8eb37ccd7d.png)

Reply:
Fixed PR as suggested ✅
I removed the unused portion of code and fixed the style of the link component, now it is identical to how it was before the change